spec/inputs/jdbc_spec.rb

```diff
@@ -894,4 +896,97 @@
       plugin.stop
     end
   end
+
+  context "when encoding of some columns need to be changed" do
+
+    let(:settings) {{ "statement" => "SELECT * from test_table" }}
+    let(:events)   { [] }
+    let(:row) do
+      {"column0" => "foo",
+       "column1" => "bar".force_encoding(Encoding::ISO_8859_1),
+       "column2" => 3}
```

Comment:
this is cosmetic nitpicking but I see you "standardized" the literal hash initialization format for the last spec context but not for the first ones. it is all valid but it is nicer to adopt just one style...